.github/workflows/migration-review.yml

```diff
@@ -0,0 +1,43 @@
+name: Migration Review
+on:
+  pull_request_target:
+    types: [opened]
+    paths:
+      - 'core/server/data/schema/**'
+      - 'core/server/data/migrations/versions/**'
+jobs:
+  createComment:
+    runs-on: ubuntu-latest
+    name: Create checklist comment
+    steps:
+      - uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            It looks like this PR contains a migration 👀
+            Here's the checklist for reviewing migrations:
+
+            ### General requirements
+
+            - [ ]  Satisfies idempotency requirement (both `up()` and `down()`)
+            - [ ]  Does not reference models
+            - [ ]  Filename is in the correct format
+            - [ ]  All early returns have appropriate log messages
```

Comment:
Can we change this to "All code paths have appropriate log messages"?

Reply:
✅